packages/react-dev-utils/FlowTypecheckPlugin.js

```diff
@@ -0,0 +1,234 @@
+var fs = require('fs');
+var path = require('path');
+var childProcess = require('child_process');
+var flowBinPath = require('flow-bin');
+const flowTypedPath = path.join(__dirname, 'node_modules', '.bin', 'flow-typed');
```

Comment:
Hi everybody! @wtgtybhertgeghgtwtg brought to my attention that decoupling `flow-typed` from `flow-typed-cli` would be a good idea... although I am helping out maintaining `flow-typed`, I am not able to do this major decisions on my own... maybe @gaearon can check back with the flow-team about this?